pkg/storage/mysql/mysql.go

```diff
@@ -121,14 +123,23 @@ func (m *MySQL) Close() {
 	m.db.Close()
 }
 
+// Read reads the set of tuples associated with store and TupleKey and returns a TupleIterator.
 func (m *MySQL) Read(ctx context.Context, store string, tupleKey *openfgav1.TupleKey) (storage.TupleIterator, error) {
```

Comment:
Let's not duplicate docs here. All the methods here are already documented in `storage`. We should do just

```
// Read please see [[storage.RelationshipTupleReader.Read]]
```